lib/builder.ts

```diff
@@ -98,74 +104,67 @@ class Builder extends EventEmitter {
     }
   }
 
+  build() {
+    // return this.buildPromise();
+    if (this._cancelationRequest) {
+      return Promise.reject(new BuilderError('Cannot start a build if one is already running'));
```

Comment:
could you make `build` an async function, and then on this line we can `throw new BuildError(/* message */)`

Reply:
fixed.